README.md

```diff
@@ -44,9 +44,12 @@ Acceptance tests apply test terraform configurations to a running AAP instance a
 
 To run acceptance tests locally, start a local AAP instance following the [docker-compose instructions for local AWX development](https://github.com/ansible/awx/blob/devel/tools/docker-compose/README.md). Create an admin user for the AAP instance and save the credentials to these environment variables:
 
+Create an admin user for the AAP instance and set the following environment variables:
```

Comment:
I don't think the user needs to be an admin - that would get them the right level of permissions, but I think we should try to go with least privilege required.  On the other hand, if the lease privilege is admin, ignore my comment and request for changes :)